### PR TITLE
feat(branding): add new branding library for easier maintenance

### PR DESCRIPTION
**Pull request by damienwebdev**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
Currently, the daff-logo isn't themeable, nor is it housed in a single library for usage across projects. 

Fixes: #219 #123 #117 #191 

## What is the new behavior?
This PR adds these capabilities.

## Does this PR introduce a breaking change?
```
[x] Yes
[] No
```

The daffio logo implementation has been removed and replaced with the one from the branding library.

## Other information

### CODE REVIEW

File: libs/branding/src/assets/daff-flower.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...11.47,0,0,0,4.34-.46C54.53,1,55.28,0,56.46.36a4,4,0,0,1,2,1.94c.8,1.31.52,2,1.14,2.48s1.72.18,3.61-.07C63.69,4.65,69.32,4,70.37,6a3.8,3.8,0,0,1,.13,1.27c.08,2.73-.62,3.39-.2,4.35.35.8.93.55,3.15,1.94,1.7,1.07,2.85,1.81,3,2.94a3.35,3.35,0,0,1-.87,2.28C74.07,21,72.42,20.84,72,22.44A13.86,13.86,0,0,0,71.91,25c0,.87,0,1.17-.2,1.4-.41.41-1.14.1-2.68-.2a8.69,8.69,0,0,0-3.54-.27,5.52,5.52,0,0,0-2.75,1.74C61,29.49,61.33,31,60.34,31.27c-1.58.49-2.83-3.11-5.35-3-1.32,0-2.49,1.08-4.15,2.54-1.46,1.28-2.05,2.26-2.74,2.07s-.53-1.1-1.48-2.27a5,5,0,0,0-2.47-1.73c-.36-.1-1.13-.25-5,1.06-2.07.69-2.1.82-2.47.73-2.16-.48-1.82-4.82-4.08-5.48-.77-.23-.8.29-3.61.67s-3.33-.07-3.48-.27c-.33-.43-.25-1.06-.14-2.14s.39-1.29.27-1.74c-.26-.94-1.52-.85-2.87-1.87-.94-.71-2.29-2.22-2-3.55s2.39-2.33,3.34-2.74,1.68-.49,2.28-1.2a3.42,3.42,0,0,0,.73-1.94c.08-1.09-.39-1.44-.46-2.54A4.08,4.08,0,0,1,27.16,5.38Z"/><path class="cls-5" d="M35.26,20.38c-.48-.56.69-2.58,2.06-4.06,4-4.3,10.24-4.44,11.89-4.41s8.24.1,11.25,4.47c1.14,1.66,1.82,4.05,1.29,4.41-.36.25-.94-.69-2.29-1.06-2-.53-2.84.89-6.13,1.42a28.11,28.11,0,0,1-4.35.11,25.12,25.12,0,0,1-3.77-.17c-3-.47-3.15-1.44-5.24-1.53C37.29,19.43,35.74,20.94,35.26,20.38Z"/></svg>
```
Comment (xelaint) [inline libs/branding/src/assets/daff-flower.svg]: Run svgo.

Don't think these are needed either (after svgo runs). `id="Layer_"` and `data-name="Layer "`

File: libs/branding/src/lib/logo/logo.component.scss [hunk 2]
```diff
@@ -0,0 +1,39 @@
+$logo-daff-width: 48%;
+$logo-flower-width: 15%;
+$logo-dil-width: 31%;
+$logo-spacer-width: 3%;
```
Comment (xelaint) [inline libs/branding/src/lib/logo/logo.component.scss]: 4%

**Comment by xelaint** [comment]
Replace `<div class="simple-footer__copyright">` with the copyright component

File: libs/branding/src/lib/logo/logo.component.scss (hunk 2 above)
Comment (damienwebdev) [reply libs/branding/src/lib/logo/logo.component.scss]: @xelaint I tried to approximate as best as I could in photoshop from the svg. These have to sum to 100%. Formula is:

```
$daff + $flower + $dil + 2*$spacer;
```

Comment (xelaint) [reply libs/branding/src/lib/logo/logo.component.scss]: Ah okay, I'll play around with it.

File: apps/daffio/src/scss/theme.scss [hunk 3]
```diff
@@ -4,7 +4,7 @@ $primary: daff-configure-palette($daff-yellow, 60);
 $secondary: daff-configure-palette($daff-purple, 60);
 $tertiary: daff-configure-palette($daff-blue, 60);
 
-$theme: daff-configure-theme($primary, $secondary, $tertiary, "light");
+$theme: daff-configure-theme($primary, $secondary, $tertiary, "dark");
```
[the remark this answers is not in the document]
Comment (damienwebdev) [reply apps/daffio/src/scss/theme.scss]: resolved in 52c3dbdf678933f8f75b35fa4658c43d4fb8841e

File: libs/branding/src/assets/daff-flower.svg (hunk 1 above)
Comment (damienwebdev) [reply libs/branding/src/assets/daff-flower.svg]: resolved in 8a7216e1c09a0aa96cd5a066151f7065b472f928 for 11% saved (.4KB). Not trememendous, but better than nothing!

**Comment by damienwebdev** [comment]
> Replace `<div class="simple-footer__copyright">` with the copyright component

Resolved in 316d5dd7f4633344be59955e60197c1cd9d2dd8e

**Comment by xelaint** [comment]
lgtm, squash and i'll approve

**Comment by damienwebdev** [comment]
@xelaint squshed.

**Comment by lderrickable** [comment]
lgtm

**Pull request merged**